R/util.R

```diff
@@ -7,3 +7,23 @@ set_names <- function(x, nms) {
   names(x) <- nms
   x
 }
+
+
+vlapply <- function(...) {
+  vapply(..., FUN.VALUE = TRUE)
+}
+
+
+vcapply <- function(...) {
+  vapply(..., FUN.VALUE = "")
+}
+
+
+data_frame <- function(...) {
+  data.frame(..., stringsAsFactors = FALSE, check.names = FALSE)
+}
+
+
+dust2_file <- function(path) {
+  system.file(path, mustWork = TRUE, package = "dust2")
```

Comment:
That `mustWork` arg name always amuses me. I might write some args `not_that_bothered_if_it_works` in some of my functions.